types/index.d.ts

```diff
@@ -0,0 +1,565 @@
+/** set when v86 is built with debug mode enabled */
+declare const DEBUG: boolean;
```

Comment:
Technically this is mutable and could be changed to false to remove some of the performance penalties. Note that it's not present in the release built.